docs/fuzzer_configuration.md

```diff
@@ -0,0 +1,109 @@
+# kAFL Fuzzer Configuration
+
+The kAFL fuzzer configuration can be changed via config files, environment variables and command line switches.
+
+The [Dynaconf](https://www.dynaconf.com/) framework is used behind the scenes for configuration management, so
+everything that you learn from Dynaconf's documentation should also be applicable for kAFL fuzzer.
+
+## Configuration files precedence
+
+All configuration files are using the `YAML` format.
+
+1. kAFL fuzzer packaged default configuration file: [`kafl_fuzzer/common/config/default_settings.yaml`](https://github.com/IntelLabs/kafl.fuzzer/blob/master/kafl_fuzzer/common/config/default_settings.yaml)
+2. System-level config file: `/etc/xdg/kAFL/settings.yaml`
+3. User-level config file: `$HOME/.config/kAFL/settings.yaml`
+4. Local config file: `$PWD/kafl.yaml`
+5. Environment variable `KAFL_CONFIG_FILE` if specified
```

Comment:
6. Environment variable overrides, e.g. KAFL_WORK_DIR

Reply:
7. Command line arguments, .e.g. --work-dir
(and then change the section header to "Configuration Sources and Precedence")